build.assets/build-package.sh

```diff
@@ -378,20 +378,20 @@ else
         --vendor "${VENDOR}" \
         --description "${DESCRIPTION} ${TYPE_DESCRIPTION}" \
         --architecture ${ARCH} \
-        --package ${OUTPUT_FILENAME} \
+        --package "${OUTPUT_FILENAME}" \
         --chdir /src/buildroot \
         --directories ${LINUX_DATA_DIR} \
         --provides teleport \
         --prefix / \
         --verbose \
-        ${CONFIG_FILE_STANZA} \
-        ${FILE_PERMISSIONS_STANZA} .
+        "${CONFIG_FILE_STANZA}" \
+        "${FILE_PERMISSIONS_STANZA}" .
 
     # copy created package back to current directory
-    cp ${PACKAGE_TEMPDIR}/*."${PACKAGE_TYPE}" .
+    cp "${PACKAGE_TEMPDIR}"/*."${PACKAGE_TYPE}" .
```

Comment:
These are pretty ugly with the multiple inline quotes and splits. I think the better idea is to wrap the entire line in one set of quotes...

```suggestion
    cp "${PACKAGE_TEMPDIR}/*.${PACKAGE_TYPE}" .
```